File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/CodyError.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class CodyError(
  val message: String,
  val cause: CodyError? = null,
  val stack: String? = null,
)

File: agent/bindings/csharp/CodyError.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class CodyError
  {
    [JsonProperty(PropertyName = "message")]
    public string Message { get; set; }
    [JsonProperty(PropertyName = "cause")]
    public CodyError Cause { get; set; }
    [JsonProperty(PropertyName = "stack")]
    public string Stack { get; set; }
  }
}
